package.json

```diff
@@ -93,6 +93,7 @@
     "next": "^12.1.6",
     "nock": "^13.0.5",
     "oidc-provider": "^7.6.0",
+    "on-headers": "^1.0.2",
```

Comment:
Shouldn't this be removed?

Reply:
I'm still using it in some test fixtures, so I moved it to `devDependencies`